presto-jdbc/src/test/java/com/facebook/presto/jdbc/TestDriver.java

```diff
@@ -242,7 +260,7 @@ public void testGetCatalogs()
     {
         try (Connection connection = createConnection()) {
             try (ResultSet rs = connection.getMetaData().getCatalogs()) {
-                assertEquals(readRows(rs), list(list("system"), list(TEST_CATALOG)));
+                assertEquals(readRows(rs), list(list("blackhole"), list("system"), list(TEST_CATALOG)));
```

Comment:
Please split this out into a separate commit: "Add blackhole connector to JDBC tests"

This will make it easier to see which test changes are directly related to `executeUpdate()` changes